sdk/src/types/globals.ts

```diff
@@ -0,0 +1,20 @@
+// had to take the "| any" way out because supporting each and every "fetch" alternative type
+// is not feasable because all have different custom properties we cannot account for and the
+// "unknown" return type is checked at runtime by the extractData function
```

Comment:
Can we make this comment sound a bit more neutral? E.g. instead of `had to take the "| any" way out ...` something like `using "| any" to ensure ...`

Also might look a bit cleaner using `/* ... */` instead of `//`?